backend/src/v1/comp_mthd_recv_cd_agcy_cd_xref/comp_mthd_recv_cd_agcy_cd_xref.controller.ts

```diff
@@ -0,0 +1,32 @@
+import { Controller, Get, Param, UseGuards } from "@nestjs/common";
+import { ApiTags } from "@nestjs/swagger";
+import { CompMthdRecvCdAgcyCdXrefService } from "./comp_mthd_recv_cd_agcy_cd_xref.service";
+import { Roles } from "../../auth/decorators/roles.decorator";
+import { Role } from "../../enum/role.enum";
+import { JwtRoleGuard } from "../../auth/jwtrole.guard";
+
+@UseGuards(JwtRoleGuard)
+@ApiTags("complaint-method-received-by")
+@Controller({
+  path: "complaint-method-received-by",
```

Comment:
Just a security concern, this api path hasn't been used elsewhere, so we might want to clean it up? cuz the api doesn't require any role (findAll, findOne)